worksheet_A/TerminalHacking.cpp

```diff
@@ -3,10 +3,42 @@
 
 #include "stdafx.h"
 #include "WordList.h"
+#include <string>
+#include <iostream>
+
+using namespace std;
 
 const int wordLength = 5;
 const int numberOfWords = 15;
 
+// Gets the similarity of the words
+int likeness(string wordGuessed, string secretWord)
+{
+	int result = 0;
+
+		for (int i = 0; i < (secretWord.length()); i++)
+		{
+		if (secretWord.at(i) == wordGuessed.at(i))
+		{
+			result ++;
+		}
+	}
+	return result;
+}
+
+//Makes user enter a word in the list or it loops
+bool listedWord(string userInput, set<string> options)
```

Comment:
`std::set` has a built-in method for checking whether it contains an element; see http://stackoverflow.com/questions/1701067/how-to-check-that-an-element-is-in-a-stdset

Note that `set::find` is O(log n) whereas your linear search is O(n), so using the built-in method would be faster for larger lists of words too.